<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>680</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1290</width>
      <height>670</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>MODBUS</string>
     </attribute>
     <widget class="QGroupBox" name="setting">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1271</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Setting</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Protocol</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Device</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbbProtocol">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>MODBUS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MSBUS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RSPBUS</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="cbbDevice">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>40</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>RELAY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AI</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="cbbCom">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ComPort</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtAddress">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>40</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="Start">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>40</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btadd">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>40</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btdelete">
       <property name="geometry">
        <rect>
         <x>1120</x>
         <y>40</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="DI_groupBox">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>120</y>
        <width>341</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>Digital Input</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="AI_groupBox">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>120</y>
        <width>351</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>Analog Input</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="Relay_groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>541</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>Relay</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>541</width>
         <height>421</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="main_gridLayout"/>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>AAA</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProtocolModBus">
    <property name="title">
     <string>ProtocolModBus</string>
    </property>
   </widget>
   <addaction name="menuProtocolModBus"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
